phobos/sys/system.d

```diff
@@ -39,6 +39,7 @@ immutable
         openBSD,   /// OpenBSD
         dragonFlyBSD, /// DragonFlyBSD
         solaris,   /// Solaris
+        haiku, /// HaikuOS
```

Comment:
Given that haiku is not supported except on ancient compilers, and not without third party patches either (gcc2.95), should it really be considered here?

Reply:
You have a good point, I don't know if you want to remove it in that case.

Reply:
I don't think it matters much either way. It's not like there's support for tvOS or watchOS.